spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -2168,7 +2168,8 @@ object QueryPlanSerde extends Logging with CometExprShim {
    */
   private def supportedSortMergeJoinEqualType(dataType: DataType): Boolean = dataType match {
     case _: ByteType | _: ShortType | _: IntegerType | _: LongType | _: FloatType |
-        _: DoubleType | _: StringType | _: DateType | _: DecimalType | _: BooleanType =>
+        _: DoubleType | _: StringType | _: DateType | _: DecimalType | _: BooleanType |
+        _: TimestampType =>
```

Comment:
Should `TimestampNTZType` also be supported?

Reply:
TimestampNTZType is supported; there is another case 1 line below.

```scala
    case TimestampNTZType => true
```